crates/matrix-sdk/src/media.rs

```diff
@@ -400,6 +400,10 @@ impl Media {
         request: &MediaRequestParameters,
         use_cache: bool,
     ) -> Result<Vec<u8>> {
+        if let Some(uri) = Self::as_local_uri(&request.source) {
```

Comment:
Can you add a small comment on top: `// Ignore request parameters for local medias, notably those pending in the send queue.`

Reply:
Added.